custom_components/tuya_local/devices/hlg_infinity_alarm.yaml

```diff
@@ -0,0 +1,332 @@
+name: HLG Infinity Alarm
```

Comment:
In case of other branded devices matching, this should be a generic name, and you should add a `products` section below this with the `id` from product_id (in tuya portal, reported by `python -m tinytuya scan`, or from the logs when using cloud-assisted setup) and the brand and model as `name`.

Reply:
This seems to be a very specific and custom project made by a Brazilian company, even the enum values are in Portuguese, that's why I think It won't be used by other companies, and that's why I made it specific. Do you still think it's a good idea to make it generic? If so I will adjust it.

Reply:
Yes, all configs have the same pattern of putting branding under products only, only some older ones have not been updated.

Reply:
Ohh ok, gotcha!

Reply:
Fixed!